src/Common/src/System/Net/Logging/GlobalLog.cs

```diff
@@ -173,41 +183,52 @@ public static void Enter(string functionName, string parameters)
             EventSourceLogging.Log.FunctionStart(functionName, parameters);
         }
 
-        public static void Assert(bool condition, string messageFormat, params object[] data)
+        public static void Assert(bool condition, string message)
```

Comment:
I added a TODO.  I couldn't entirely remove these until the rest of System.Net.* is off of them, and there's a bunch of work (similar to what I'm doing in this PR) for other System.Net libs to remove similar overheads.  I've updated https://github.com/dotnet/corefx/issues/5144 with more tracking items.  This PR at least ensures that the remaining few call sites for these Assert(bool, ...) methods don't do any unnecessary work at the call sites.